internal/client.go

```diff
@@ -835,6 +839,33 @@ type (
 		NonRetryableErrorTypes []string
 	}
 
+	// Priority contains metadata that controls the relative ordering of task processing
+	// when tasks are backed up in a queue. The affected queues depend on the
+	// server version.
+	//
+	// Priority is attached to workflows and activities. By default, activities
```

Comment:
@stephanos based of our previous conversation https://github.com/temporalio/sdk-go/pull/1792/files#r1934130657 should this say

> By default, activities and child workflows inherit Priority from the workflow that created them

?

Reply:
I agree, that makes it more explicit 👍